data/square-disc-nurbs-patch.mesh

```diff
@@ -0,0 +1,155 @@
+MFEM NURBS mesh v1.0
```

Comment:
This same file is duplicated in `miniapps/nurbs`

Reply:
Removed